catalog/airflow/x-definitions/app-airflow.cue

```diff
@@ -246,8 +246,29 @@ template: {
 				mysqlSecret: string
 			}
 		}
-		// +ui:title=Webserver
+
+		// ui:title=DAG 配置
 		// +ui:order=2
+		dags: {
+			// +ui:description=git 同步配置 (必须配置)
+			gitSync: {
+				// +ui:description=git 仓库地址
+				// +ui:order=1
+				repo: *"https://gitee.com/linktime-cloud/example-datasets.git" | string
```

Comment:
We should use GitHub repo for default value, you may add Gitee mirror repo in the description.

Reply:
updated